<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>247</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="FTP20_BLOCK">
        <property name="text">
         <string>FTP(20) 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="FTP21_BLOCK">
        <property name="text">
         <string>FTP(21) 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HTTP_BLOCK">
        <property name="text">
         <string>HTTP 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HTTPS_BLOCK">
        <property name="text">
         <string>HTTPS 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IMAP_BLOCK">
        <property name="text">
         <string>IMAP 차단</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="NETBIOS_BLOCK">
        <property name="text">
         <string>NETBIOS 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="WINDOWS_REMOTE_BLOCK">
        <property name="text">
         <string>윈도우 원격제어 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SNMP_BLOCK">
        <property name="text">
         <string>SNMP 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SSH_BLOCK">
        <property name="text">
         <string>SSH 차단</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_15">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>102</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="TCP_ALL_BLOCK">
      <property name="text">
       <string>TCP 전체차단</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="UDP_ALL_BLOCK">
      <property name="text">
       <string>UDP 전체차단</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
